### ISSUE TITLE
Advice on Which Metabolic Parameters Ought to be Measured Next

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#479]
A set of compounds has been shipped to the [CDCO](https://www.monash.edu/pharm/research/areas/optimisation) at Monash University for some _in vitro_ metabolic work that is much-needed in Series 4. **Question**: what needs measuring on these compounds?

The relevant set of compounds is as follows (spreadsheet appended below).

![monash consult pic](https://cloud.githubusercontent.com/assets/4386101/22360605/715460b2-e4a5-11e6-9fec-83a84df8e2d0.png)

The main box in the figure contains the Frontrunners (#400), or close analogs thereof. There are also some new analogs that are more soluble than most Series 4 compounds. Plus some amides evaluated in the Kirk assay (re competition in #421 ).

What is known of the Series 4 compounds’ metabolic clearance rates and solubility is [here](http://www.openwetware.org/wiki/OpenSourceMalaria:Triazolopyrazine_(TP)_Series#Physicochem.2Fmetabolism.2FPK). A summary of what’s known for the Frontrunner set is [here](https://cloud.githubusercontent.com/assets/2626599/15854939/f3b4d01e-2cef-11e6-932c-8c1b07c5c413.png).

What do we ask CDCO to measure? We need data both strategically (for judging the series) and for the 1st Series 4 paper (i.e. sometimes we need neat datasets, just because).

My questions:

1) We need a complete set of microsomal data for _in vitro_ clearance rate (CLint) and half life, but for mouse, rat or human? Or a combination? For completeness (i.e. vs existing data) I would suggest human and mouse.

2) Should we yet go for hepatocyte data (again, Clint and half life), and if so, mouse or human? For completeness I would suggest human and rat.

3) Presumably we need the LogD values at both pH’s (3.0 and 7.4) and ditto solubility measurements (pH 2 and pH 6.5)? I see this as essential for adding to our predictive understanding of solubility in this series, when we’re designing new compounds.

Later, with these data in hand, we need to be selecting 3 compounds for _in vivo_ evaluation to answer the IVIVC question once and for all (#389).

Looking again at the frontrunner data I am particularly interested in seeing the metabolic behavior of the promising compounds MMV672687 and MMV670437 (the latter was supposedly enantioenriched, but here is tested as the unequivocally racemic version, MMV897763). Remember for a late lead we need i) potency <100 nM, and ii) ideally solubility *>100uM* (40 microg/mL if assume MW ca. 400)

[Monash Compounds.xlsx](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/734398/Monash.Compounds.xlsx)

**Comment by PaulWillisMMV** [comment]
1. I would support 1 rodent and 1 human in vitro measurement and would select  rat heps and human mics: We will use rat as the PK species so for the IVIVE we need a rat in vitro measure.  Human mics then allows us an initial estimate of whether we have made any improvements in the human metabolic stability.  All data to date suggests that the series has poor human stability so running all in human heps is not necessary, human mics will indicate possible improvements .  I would not worry about mouse stability at this stage: we are not going to use the data for IVIVE.  We can always go back and collect additional data on the most promising compounds
2. I would just support Log D 7.4 and a PBS solubility measurement.  Solubility and Log D measurements at pH3 are not needed.
3.  I would not collect in vitro DMPK or phys chem data on any compounds with a potency >500nM UNLESS the compounds really explores a different property space.  None of the inactive compound seem to meet this criteria
4. MMV897709 does not need in vitro clints just Log D and solubility - we already have Clints on the enantiomer MMV669844
5. I would get rat heps on MMV67024 if we do not have it (missing from table) - we have the rat PK so this provides another IVIVE point
6. I cant see the structure or data on MMV897763 so cant comment on its interest
7. Some of the front runners already seem to have the required data - no need to repeat
8.  There is a typo - late leads should have a potency <10nM (not <100nM)
http://www.mmv.org/sites/default/files/uploads/docs/essential_info_for_scientists/frontrunner_templates/2014_Late_Lead_-_Frontrunner_template.xlsx

**Comment by MFernflower** [comment]
I would be curious to see what happens to the rac-methoxy compound when it goes liver-pass

**Comment by drc007** [comment]
@mattodd There are a few things to consider when planning metabolism studies.
When thinking about species, you need data for man (your clinical species), your safety species (usually rat and dog), and your species used for efficacy studies if different from the safety species. At this stage in the program rat and human will probably suffice.

Microsomes usually give more reproducible data (less batch to batch variation) but lack some key cytosolic enzymes such as aldehyde oxidase. If you think this might be important then hepatocytes may be more useful. If you want to use the data for structure activity you need all the compounds run under the same assay protocol.

You only need LogD at pH 7.4, given the pKa of the molecule you can then calculate the LogD at other pH if needed.

We are looking for a significant improvement in solubility, looking at a single pH should give you that insight.

**Comment by mattodd** [comment]
Thanks for the inputs. Measurements now done (#484)

Ultimately the 18 compounds were measured for:

i) mouse/human microsomal clearance
ii) kinetic solubility (pH 2 and 6.5)
iii) chromatographic Log D at pH 7.4
iv) chromatographic protein binding

This is obviously very close to what we had been discussing here. Paul suggested human mic and rat heps, so the difference is that we got mouse mic rather than rat hep. The new data are a good fit with what has been obtained before, which will help with publication. I suggest we zoom in on rat hep for any further studies related specifically to IVIVC.

We got LogD at 7.4, and kinetic solubility at pH 2 and 6.5, rather than 7.4. Again this is a better fit with existing data for publication and to help with solubility predictions. Solubility for poorly-active compounds is still useful to establish the predictive utility of our cLogP calculations for this series. Similarly clearance for some of the low-active compounds might help explain poor activities, particularly with respect to Kiaran’s assay. These are more for the paper than the compound progression, and again, once we have these data for this set we have what we need for the paper.

The racemic sample MMV897709 needs measurement because the so-called enantiopure sample MMV669844 has no proof of it being enantiopure. We’re establishing the difference once and for all through resolution of that compound which has, I understand, been successful (#470).

MMV670246 rat heps. Yes this is important. We should include this in any further assessments, and this was actually already flagged in the IVIVC planning (#389).

MMV897763 is the compound towards the right of the box in the figure. The promising dimethylamine. When the OCHF2 was replaced with CN in the northeast we lost all activity, which has been one of the most dramatic cliffs we’ve seen in the series. Some of the new measurements might reveal why.

Agreed that some of the frontrunners already have some of the data. Several people have commented in the past on the desirability of screening the whole set in one assay for comparison for the paper, which is what’s been done here. The new data will be folded in with the old.

The measurements of protein binding were one of the strongest suggestions to come from Mark’s last community webinar (#394), and were mentioned as desirable by the referees on our grant last time around. Hopefully useful. Closing now, and directing further discussion to #484.
